### ISSUE TITLE
tipi cannot be installed by default on a fresh macOS

### ISSUE THREAD

**Issue by daminetreg** [tipi-build/cli#59]
The python installed by default on macbooks are 2.7 and have an outdated certstore. The python installed by tipi on macOS fails with : 

```
▶ Shipping tool with name 'emsdk'
initializing emsdk.
`update-tags` is not longer needed.  To install the latest tot release just run `install tot`
Installing SDK 'sdk-releases-upstream-89202930a98fe7f9ed59b574469a9471b0bda7dd-64bit'..
Installing tool 'node-14.15.5-64bit'..
Error: Downloading URL 'https://storage.googleapis.com/webassembly/emscripten-releases-builds/deps/node-v14.15.5-darwin-x64.tar.gz': <urlopen error unknown url type: https>
Warning: Possibly SSL/TLS issue. Update or install Python SSL root certificates (2048-bit or greater) supplied in Python folder or https://pypi.org/project/certifi/ and try again.
Installation failed!
● [err] failed.
● [err] Dynamic exception type: std::runtime_error
std::exception::what: CTest failed
● [err] Something went wrong, for help on your issue, we would enjoy getting your report at https://tipi.build/#contact
➜  ~ Error: Downloading URL 'https://storage.googleapis.com/webassembly/emscripten-releases-builds/deps/node-v14.15.5-darwin-x64.tar.gz': <urlopen error unknown url type: https>
Warning: Possibly SSL/TLS issue. Update or install Python SSL root certificates (2048-bit or greater) supplied in Python folder or https://pypi.org/project/certifi/ and try again.
Installation failed!
➜  ~ Error: Downloading URL 'https://storage.googleapis.com/webassembly/emscripten-releases-builds/deps/node-v14.15.5-darwin-x64.tar.gz': <urlopen error unknown url type: https>
Warning: Possibly SSL/TLS issue. Update or install Python SSL root certificates (2048-bit or greater) supplied in Python folder or https://pypi.org/project/certifi/ and try again.
Installation failed!
```

**Comment by daminetreg** [comment]
**Workaround :** disable emsdk in the distro.json

Store the file distro.json on the disk, then run

```
export TIPI_DISTRO_JSON=/path/to/distro.json
```

Ensure the following content is in the distro : 
```
{
  "modes" : { 
    "default" : ["python", "openssh", "environments", "ssh-over-http-proxy", "elfshaker"]
  }

  ,"distro": [
    {
      "name": "cmake",
      "platforms": {
        "macos": {
          "x86_64": {
            "url": "https://nxxm.indigenious.io/distro/all/CMake/v3.21.4/cmake-3.21.4-Darwin-x86_64.zip",
            "sha1": "1b1b5c6b0db43541f3fa9beaf0885ad52eb63f4f",
            "root": "cmake-3.21.4-Darwin-x86_64/CMake.app/Contents",
            "module_path": "share/cmake-3.21/Modules",
            "path": ["bin"]
          } 
        },
        "linux": {
          "x86_64": {
            "url": "https://nxxm.indigenious.io/distro/all/CMake/v3.21.4/cmake-3.21.4-linux-x86_64.zip",
            "sha1": "aa668b36aa3e70409a2295e448217f95056f082e",
            "root": "cmake-3.21.4-linux-x86_64",
            "module_path": "share/cmake-3.21/Modules",
            "path": ["bin"]
          }
        },
        "windows": {
          "x86_64": {
            "url": "https://nxxm.indigenious.io/distro/all/CMake/v3.21.4/cmake-3.21.4-win64-x64.zip",
            "sha1": "78da891a6080655bb93b97d9b1c09d6fa8f5b264",
            "root": "cmake-3.21.4-win64-x64",
            "module_path": "share/cmake-3.21/Modules",
            "path": ["bin"]
          }
        }
      }
    },
    {
      "name": "make",
      "platforms": {
        "windows": {
          "x86_64": {
            "url": "https://nxxm.indigenious.io/distro/v0.0.12/make/mingw32-make.zip",
            "sha1": "808948b4cbf6c3f1812cb5922e926fe8479e7be3",
            "root": "mingw32-make",
            "path": ["."]
          }
        },
        "linux": {
          "universal": {
            "url": "",
            "sha1": "",
            "root": ""
          }
        },
        "macos": {
          "universal": {
            "url": "",
            "sha1": "",
            "root": ""
          }
        }
      }
    },
  
    {
      "name": "environments",
      "platforms": {
        "all": {
          "universal": {
            "url": "https://github.com/tipi-build/environments/archive/d40aa228196a08d0260454cdb8ba93b8f3c02518.zip",
            "sha1": "33dc11ff04a796f13c74de07b203f9301472376e",
            "root": "environments-d40aa228196a08d0260454cdb8ba93b8f3c02518"
          }
        }
      }
    },
  
    {
      "name": "perl",
      "platforms": {
        "windows": {
          "x86_64": {
            "url": "https://nxxm.indigenious.io/distro/v0.0.12/perl/strawberry-perl-5.30.3.1-windows-64bit.zip",
            "sha1": "ac3ce102081ac6f9b3b43d81c30ac77b08493eff",
            "root": "",
            "path": ["perl/bin"]
          }
        }
      }
    },
  
    {
      "name": "python",
      "platforms": {
        "macos": {
          "x86_64": {
            "url": "https://nxxm.indigenious.io/distro/v0.0.12/python/python-3.8.0-Darwin-x64.zip",
            "sha1": "54b633011cd24867662c9dbaeda596bb3261a09c",
            "root": "",
            "path": [".", "bin"]
          }
        },
        "linux": {
          "x86_64": {
            "url": "https://github.com/tipi-build/cpython-build/releases/download/3.10.5p1/tipi-python-3.10.5-w-openssl-1.1.1o.zip",
            "sha1": "322db30c71f259efff38c687e5dd5f9b1e92fb9e",
            "root": "",
            "path": ["bin"]
          }
        },
        "windows": {
          "x86_64": {
            "url": "https://nxxm.indigenious.io/distro/all/python/python-3.7.4-embed-amd64.zip",
            "sha1": "ac02febae0898572a7353be909448bd1acac4a79",
            "root": "",
            "path": [".", "bin"]
          }
        }
      }
    },
  
    {
      "name": "nodejs",
      "platforms": {
        "macos": {
          "x86_64": {
            "url": "https://nxxm.indigenious.io/distro/v0.0.12/nodejs/node-v14.7.0-darwin-x64.zip",
            "sha1": "71c4c7ceddafd46143832abc136c592192c70713",
            "root": "node-v14.7.0-darwin-x64",
            "path": [".", "bin"]
          }
        },
        "linux": {
          "x86_64": {
            "url": "https://nxxm.indigenious.io/distro/v0.0.12/nodejs/node-v14.7.0-linux-x64.zip",
            "sha1": "4472af28a2bd943fe64f4dfb9d44c5ebde152ec0",
            "root": "node-v14.7.0-linux-x64",
            "path": [".", "bin"]
          }
        },
        "windows": {
          "x86_64": {
            "url": "https://nxxm.indigenious.io/distro/v0.0.12/nodejs/node-v14.7.0-win-x64.zip",
            "sha1": "ec44ef72df91ffc4b02df49e75c46b8f845fab5e",
            "root": "node-v14.7.0-win-x64",
            "path": [".", "bin"]
          }
        }
      }
    },
    {
      "name": "emsdk",
      "platforms": {
        "all": {
          "universal": {
            "url": "https://github.com/emscripten-core/emsdk/archive/7cba2e0a92b9df42376707dfdae6e1f108f79a27.zip",
            "sha1": "f693e7f511d2d320d20a1376812321763027728e",
            "root": "emsdk-7cba2e0a92b9df42376707dfdae6e1f108f79a27",
            "path": ["bin", "upstream/emscripten", "python/3.7.4-pywin32_64bit"]
          }
        }
      }
    },
  
    {
      "name": "hunter",
      "platforms": {
        "all": {
          "universal": {
            "url": "https://github.com/nxxm/hunter/archive/6efe3552dd0c1a547e8a18067003be7222d0cb24.zip",
            "sha1": "6b9afd6a4d5cd916ea1d1dfd3adea3590bace56d",
            "root": "hunter-6efe3552dd0c1a547e8a18067003be7222d0cb24",
            "target": "platform"
          }
        }
      }
    },
  
    {
      "name": "hunter_gate",
      "platforms": {
        "all": {
          "universal": {
            "url": "https://github.com/hunter-packages/gate/archive/42c1c27cdd556dcbc358c6ba8ad7a72a68ee2a62.zip",
            "sha1": "a72c0958bdfdb4666ee077cdd40c22875a082915",
            "root": "gate-42c1c27cdd556dcbc358c6ba8ad7a72a68ee2a62/cmake/HunterGate.cmake",
            "relative_dest_file": "HunterGate.cmake"
          }
        }
      }
    },
    {
      "name": "clang",
      "platforms": {
        "macos": {
          "universal": {
            "url": "https://nxxm.indigenious.io/distro/all/clang/v13.0.0/clang-13-darwin-64bit.zip",
            "sha1": "a7e6968a4de8a8528349ec3291156f2b15e8767f",
            "root": "clang-macos",
            "path": ["bin"]
          }
        },
        "linux": {
          "universal": {
            "url": "https://nxxm.indigenious.io/distro/all/clang/v13.0.0/clang-13-linux-1604-64bit.zip",
            "sha1": "4f846ee40c9ae0f6182fd7553ef8684ff06e63bc",
            "root": "clang-linux-1604",
            "path": ["bin"]
          }
        },
        "windows": {
          "universal": {
            "url": "https://nxxm.indigenious.io/distro/all/clang/v13.0.0/clang-13-windows-64bit.zip",
            "sha1": "6f9b4e0b86ae48a818fe95073acc5be05839a5e4",
            "root": "mingw64-clang",
            "path": ["bin"]
          }
        }
      }
    },
    {
      "name": "go",
      "platforms": {
        "macos": {
          "universal": {
            "url": "https://nxxm.indigenious.io/distro/v0.0.12/go/go_1.14.14_darwin_64bit.zip",
            "sha1": "1e2561d8307eb513a23eb7c9a8b769b800db27af",
            "root": "go",
            "path": ["bin"]
          }
        },
        "linux": {
          "universal": {
            "url": "https://nxxm.indigenious.io/distro/v0.0.12/go/go_1.14.14_linux_64bit.zip",
            "sha1": "9fc8f520528b3f353b1feeb1279f241d2a5c3677",
            "root": "go",
            "path": ["bin"]
          }
        },
        "windows": {
          "universal": {
            "url": "http://nxxm.indigenious.io/distro/v0.0.12/go/go_1.14.14_windows_x86_64.zip",
            "sha1": "782c595c1aece9fea8bfa7a14bfbf52e206a0f91",
            "root": "go",
            "path": ["bin"]
          }
        }
      }
    },
    {
      "name": "openssh",
      "platforms": {
        "windows": {
          "universal": {
            "url": "http://nxxm.indigenious.io/distro/all/openssh/OpenSSH-Win64.zip",
            "sha1": "0a60a7d9d6ac67dfe4b97f720ad780dd185ed432",
            "root": "OpenSSH-Win64",
            "path": ["."]
          }
        }
      }
    },
    {
      "name": "jdk",
      "platforms": {
        "macos": {
          "universal": {
            "url": "https://nxxm.indigenious.io/distro/all/jdk-16_0_1/jdk-16.0.1-macos.zip",
            "sha1": "399c189d3145c6f3d7ab8e829bfd0d61eb30a3c5",
            "root": "jdk-16.0.1.jdk",
            "path": ["Contents/MacOS","Contents/Home/bin","Contents/Home/lib"]
          }
        },
        "linux": {
          "universal": {
            "url": "https://nxxm.indigenious.io/distro/all/jdk-16_0_1/jdk-16.0.1-linux.zip",
            "sha1": "1946904e4d5ad74f3193d07d7c8a98136dbaff74",
            "root": "jdk-16.0.1",
            "path": ["bin","lib"]
          }
        },
        "windows": {
          "universal": {
            "url": "https://nxxm.indigenious.io/distro/all/jdk-16_0_1/jdk-16.0.1-windows.zip",
            "sha1": "112dbf189260a647842e98671516f18e4de070f9",
            "root": "jdk-16.0.1",
            "path": ["bin","lib"]
          }
        }
      }
    },
    {
      "name": "openapi",
      "platforms": {
        "all": {
          "universal": {
            "url": "http://nxxm.indigenious.io/distro/all/openapi_generator/openapi-generator-5-1-1.zip",
            "sha1": "ec7cbaed034d7aaeb2a0f22619a8927c08a74894",
            "root": "openapi_generator",
            "path": ["."]
          }
        }
      }
    },
    {
      "name": "openapi-generator-script",
      "platforms": {
        "all": {
          "universal": {
            "url": "https://github.com/nxxm/openapi-generator-script/archive/77c837ffdd7331fb87be3e76deb53028d0d4cb66.zip",
            "sha1": "4fc850d4a145be49a9aa0428dcbc5c5c82f19f93",
            "root": "openapi-generator-script-77c837ffdd7331fb87be3e76deb53028d0d4cb66",
            "path": ["."]
          }
        }
      }
    },
    {
      "name": "ssh-over-http-proxy",
      "platforms": {
        "macos": {
          "universal": {
            "url": "https://github.com/tipi-build/ssh-over-http-proxy/releases/download/v1.0.0/ssh-over-http-proxy-macos.zip",
            "sha1": "bed47ab90b0d4a539af43918a6e4061b31e1da8e",
            "root": "ssh-over-http-proxy-macos",
            "path": ["."]
          }
        },
        "linux": {
          "universal": {
            "url": "https://github.com/tipi-build/ssh-over-http-proxy/releases/download/v1.0.0/ssh-over-http-proxy-linux.zip",
            "sha1": "fa9e251a9ca55c0d56ec6fe1f377267b4c1ac6af",
            "root": "ssh-over-http-proxy-linux",
            "path": ["."]
          }
        },
        "windows": {
          "universal": {
            "url": "https://github.com/tipi-build/ssh-over-http-proxy/releases/download/v1.0.0/ssh-over-http-proxy-win.zip",
            "sha1": "b1f642db4ba0155868450c2ca69508eca1c07aba",
            "root": "ssh-over-http-proxy-win",
            "path": ["."]
          }
        }
      }
    },
    {
      "name": "elfshaker",
      "platforms": {
        "macos": {
          "universal": {
            "url": "https://github.com/tipi-build/elfshaker/releases/download/v0.0.10/elfshaker-macos.zip",
            "sha1": "018c7fdc49062a29526b581f6acff0d358147583",
            "root": "",
            "path": ["."]
          }
        },
        "linux": {
          "universal": {
            "url": "https://github.com/tipi-build/elfshaker/releases/download/v0.0.10/elfshaker-linux.zip",
            "sha1": "dcafce08ad46e88977d4510db190ecd24962dcf0",
            "root": "",
            "path": ["."]
          }
        },
        "windows": {
          "universal": {
            "url": "https://github.com/tipi-build/elfshaker/releases/download/v0.0.10/elfshaker-win.zip",
            "sha1": "7eeb373f1bd24dc756d6ec7dce330b683254900d",
            "root": "",
            "path": ["."]
          }
        }
      }
    }
  ]
}
```

**Comment by daminetreg** [comment]
That has been fixed since version [v0.0.51](https://github.com/tipi-build/cli/blob/master/CHANGELOG.md#v0051---nitfy-nautilus-)
